### PR TITLE
[TESTMERGE] Properly working TGUI Next

### PR DESCRIPTION
**Pull request by Arturlang**
This PR ports TGUI next from TG, which was added with a ton of PRs
Mainly by https://github.com/actioninja and https://github.com/stylemistake
## Changelog
:cl:
add: Replaces a lot of ingame UIs with TGUI next, increasing performance drastically.
/:cl:
This doesnt remove TGUI or oracleui, itll have to be done in a separate PR, and oracleui is completely unused at the moment, and someone needs to do TGUI next for some interfaces that were modified.

### CODE REVIEW

**Comment by IronEleven** [comment]
Gas mixers aren't working.  You set a node to any percentage and it becomes 100

**Comment by IronEleven** [comment]
The nuke countdown is an hour by default.

Edit: It's not an hour by default, the interface is just confusing

**Comment by Arturlang** [comment]
@IronEleven The interface takes some getting used to, will check the gas mixers

File: tgui-next/packages/tgui/interfaces/Cargo.js [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+              key={entry.id}
+              className="candystripe"
+              label={'#' + entry.id}
+              buttons={(
+                <Fragment>
+                  <Box inline mx={2}>
+                    {!!entry.paid && (<b>[Paid Privately]</b>)}
+                    {' '}
+                    {entry.cost} credits
+                  </Box>
```
Comment (LetterN) [inline tgui-next/packages/tgui/interfaces/Cargo.js:256]: you should remove this for now and recompile tgui-next, we don't have currency on this server

File: tgui-next/packages/tgui/public/tgui-main.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        /:::/    /                               \::/____/              \::::/    /
+        \::/    /                                                        \::/    /
+         \/____/                                                          \/____/
+</div>
+<marquee class="FatalError__header">
+A fatal exception has occurred at 002B:C562F1B7 in TGUI.
+The current application will be terminated.
+Please remain calm, get to the nearest NTNet workstation
+and send the copy of the following stack trace to:
+www.github.com/tgstation/tgstation. Thank you for your cooperation.
```
Comment (LetterN) [inline tgui-next/packages/tgui/public/tgui-main.html]:
```suggestion
https://github.com/tgstation/tgstation Thank you for your cooperation.
```

File: code/game/machinery/computer/atmos_control.dm [hunk 3]
```diff
@@ -280,13 +293,19 @@ GLOBAL_LIST_EMPTY(atmos_air_controllers)
...
 		if("output")
 			signal.data += list("tag" = output_tag, "power_toggle" = TRUE)
 			. = TRUE
 		if("pressure")
-			var/target = input("New target pressure:", name, output_info ? output_info["internal"] : 0) as num|null
-			if(!isnull(target) && !..())
-				target =  CLAMP(target, 0, 50 * ONE_ATMOSPHERE)
+			var/target = text2num(params["pressure"])
+			if(!isnull(target))
+				target = CLAMP(target, 0, 4500)
```
Comment (LetterN) [inline code/game/machinery/computer/atmos_control.dm]:
```suggestion
				target = CLAMP(target, 0, MAX_OUTPUT_PRESSURE)
```

File: tgui-next/packages/tgui/interfaces/Cargo.js (hunk 1 above)
Comment (Arturlang) [reply tgui-next/packages/tgui/interfaces/Cargo.js:256]: Been trying to do that, but having some problems with indendation, really my first forray with javascript.

File: code/__DEFINES/atmospherics.dm [hunk 4]
```diff
@@ -149,9 +149,9 @@
 
 //OPEN TURF ATMOS
 #define OPENTURF_DEFAULT_ATMOS		"o2=22;n2=82;TEMP=293.15" //the default air mix that open turfs spawn
-#define TCOMMS_ATMOS				"n2=100;TEMP=80" //-193,15�C telecommunications. also used for xenobiology slime killrooms
```
Comment (Putnam3145) [inline code/__DEFINES/atmospherics.dm]: please fix whatever the hell is going on here, this diff is doing literally nothing but causing spurious merge conflicts

**Review by Putnam3145 — DISMISSED** [review]

**Review by Ghommie — APPROVED** [review]

**Pull request merged**